__tests__/endpoint.test.js

```diff
@@ -39,4 +39,55 @@ describe('GET /api/topics', () => {
             expect(body.msg).toBe('404: Page Not Found.')
         } )
     })
+})
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('status:200, responds with votes increased by amount added', () => {
+        const articleUpdates = {
+            inc_votes: 3
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 103 })
+        })
+    })
+    test('Works with negative numbers', () => {
+        const articleUpdates = {
+            inc_votes: -50
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 50})
+        })
+    })
+    test('If the number would go below 0, it remains at 0', () => {
+        const articleUpdates = {
+            inc_votes: -500
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 0})
+        })
+    } )
+    test('If input is NaN, return an error and a sarcastic message.', () => {
+        const articleUpdates = {
+            inc_votes: 'Im an idiot'
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(404)
+        .then(({body}) => {
+            expect(body).toEqual({msg: 'Is it that hard to just use numbers?'})
```

Comment:
It's really not... but it might make the user feel bad if you tell them this 😂

Reply:
also would this be a 400 or a 404? 
is it a bad request or is the id just not found?